tools/gcsbackup/README.md

```diff
@@ -1,19 +0,0 @@
-# GCSbackup
```

Comment:
Where do you think it should go? Note that I moved the GCSBackup script into the content directory, which has several unrelated scripts as of now.